ocaml-lsp-server/src/client.ml

```diff
@@ -0,0 +1,54 @@
+open Import
+
+module Vscode = struct
```

Comment:
This module is missing `client.mli`. All the useful comments would be better placed there.